src/document_to_podcast/inference/text_to_speech.py

```diff
@@ -40,33 +88,66 @@ def _text_to_speech_parler(
     return waveform
 
 
-def text_to_speech(
-    input_text: str,
-    model: Union[InterfaceGGUF, PreTrainedModel],
-    voice_profile: str,
-    tokenizer: PreTrainedTokenizerBase = None,
+def _text_to_speech_parler_multi(
```

Comment:
Am i missing something or are this and the indic function the same?

Reply:
there is a tiny difference here

```
    generation = model.generate(input_ids=input_ids, prompt_input_ids=prompt_input_ids)
```

vs
```
    generation = model.generate(
        input_ids=input_ids.input_ids,
        attention_mask=input_ids.attention_mask,
        prompt_input_ids=prompt_input_ids.input_ids,
        prompt_attention_mask=prompt_input_ids.attention_mask,
    )
```
but I should probably rewrite it, its too much duplicate code